web_src/js/utils/dom.js

```diff
@@ -41,16 +21,10 @@ function elementsCall(el, func, ...args) {
 function toggleShown(el, force) {
   if (force === true) {
     el.classList.remove('gt-hidden');
-    assertShown(el, true);
   } else if (force === false) {
     el.classList.add('gt-hidden');
-    assertShown(el, false);
   } else if (force === undefined) {
-    const wasShown = window.config.runModeIsProd ? undefined : isShown(el);
     el.classList.toggle('gt-hidden');
```

Comment:
Can't that be simplified now to `el.classList.toggle('gt-hidden', force)`?

Reply:
It probably can but I guess it may be slower then explit add/remove. But would need to check how browsers implement toggle.

Reply:
I think maybe one day we should make it work with official `hidden` attribute together, because the `[hidden]` is standard.

Then we need to remove (in show) and check (in toggle) the `hidden` attribute. Then such `if` blocks would help.